sdk/webpubsub/azure-messaging-webpubsubservice/samples/send_messages_aad.py

```diff
@@ -49,7 +49,7 @@
 # Send a json message to everybody on the given hub...
 try:
     # Raise an exception if the service rejected the call
-    client.send_to_all('Hub', message={'Hello': 'all'})
+    client.send_to_all('Hub', message={'Hello': 'all'}, content_type='application/json')
```

Comment:
I don't think you actually need to specify content type here, it should default to application/json